### ISSUE TITLE
Maintainer wanted

### ISSUE THREAD

**Issue by lucax88x** [lucax88x/CodeAceJumper#439]
As title says, unfortunately I don't use vscode anymore, therefore I'll be glad to pass this repository to someone who wants to add feature / maintain it.

**Comment by LGala** [comment]
@lucax88x Did you find someone? I really like this extension, I can mantain It.

**Comment by lucax88x** [comment]
Yeah sure. It's yours.

Try to take some confidence with this code maybe.. it's starting to have its age.

I can help in the process ;)

**Comment by LGala** [comment]
@lucax88x Thanks. You need to do something, like transfer the ownership, and so on?

BTW: thanks for your help, I think I'll start hacking with the code locally, open PRs to solve some open issues, adding some tests (if needed, but I think tests are never enough) and then ultimately I'll refactor it if needed.

What do you think?

**Comment by LGala** [comment]
Ehi @lucax88x don't want to put any pressure. I'm not used to this kind of things and I'm just asking if you need to do something. Thanks!

**Comment by lucax88x** [comment]
Sorry :( was a bit busy with job.

I think it's a great idea. I remember I added some e2e tests but Idk if they still work. I can help on reviewing PRs etc.. but cannot promise anything.

About ownership we sure can do that, I think that first you can just try like you said, doing PRs, tests, etc... then if you still feel engaged we can talk about that!

**Comment by LGala** [comment]
Yeah, yeah no problem! I thought you just missed the comments, no worries.

Ok! I will do as said! Thanks for your time. Have a nice weekend
